.github/workflows/test-cron.yaml

```diff
@@ -253,18 +246,34 @@ jobs:
       run: './pants --version
 
         '
+    - name: Run smoke tests
+      run: './pants list ::
+
+        ./pants roots
+
+        ./pants help goals
+
+        ./pants help targets
+
+        ./pants help subsystems
+
+        '
+    - if: always()
+      name: Upload pants.log
+      uses: actions/upload-artifact@v3
+      with:
+        name: pants-log-bootstrap-macOS-x86_64
+        path: .pants.d/pants.log
     - name: Upload native binaries
       uses: actions/upload-artifact@v3
       with:
-        name: native_binaries.${{ matrix.python-version }}.${{ runner.os }}
+        name: native_binaries.${ matrix.python-version }.macOS-x86_64
         path: '.pants
 
           src/python/pants/engine/internals/native_engine.so
 
           src/python/pants/engine/internals/native_engine.so.metadata'
-    - env:
-        TMPDIR: ${{ runner.temp }}
```

Comment:
Moved to the global env, instead of being applied haphazardly on specific jobs/steps.